core/src/main/scala/akka/kafka/ConsumerSettings.scala

```diff
@@ -172,6 +153,7 @@ object ConsumerSettings {
   /**
```

Comment:
Yes, we could add guard for null values. What is the best way? 

1.Change `Some` to `Option`.

`apply(system, Option(keyDeserializer), Option(valueDeserializer))`

null values would be `None` and it would be validated in the other `apply` method. We have nothing more to change.

2.Add specific validation as you comment.

I vote for 1 because for `create` methods that also calls `apply` using `scala-java8-compat`:

`apply(config, keySerializer.asScala, valueSerializer.asScala)`

`asScala` is converting `null` values to `None`.